src/components/Charts/ChataChart/ChataChart.js

```diff
@@ -192,9 +191,35 @@ export default class ChataChart extends Component {
     return { chartHeight, chartWidth }
   }
 
+  sortChartDataByDate = (props) => {
+    try {
+      if (!props.data || typeof props.data !== 'object') {
+        return undefined
```

Comment:
Instead of returning undefined we should probably return the original `props.data` just to keep it the same going in as going out